backend/main.py

```
@@ -0,0 +1,47 @@
+def health_check():
+    """Health check endpoint - returns API status"""
+    return {
+        "status": "healthy",
+        "message": "API is running",
+        "version": "1.0.0"
+    }
+
+if __name__ == "__main__":
+    uvicorn.run(
+        "main:app",
+        host="0.0.0.0",
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)